collect_app/src/test/java/org/odk/collect/android/widgets/items/SelectMultiWidgetTest.java

```diff
@@ -226,6 +207,66 @@ public void whenSpacesInUnderlyingValuesExist_shouldAppropriateWarningBeDisplaye
         assertThat(warningTv.getText(), is("Warning: underlying values a a, b b have spaces"));
     }
 
+    @Test
+    public void checkingCheckboxOnItem_callsValueChangedListener() {
```

Comment:
It would be good to add the same tests for `SelectOneWidgetTest`. That widget worked well but I think we don't have such tests there to make sure everything will be fine in the future.